docs/maintainer.md

```diff
@@ -8,6 +8,21 @@ Helping maintain a project is the best way to contribute to its overall health.
 
 ### Triaging Issues
 
+We encourage the community to help make bugs actionale, make features available,
+ and close stale issues. Triaging is one of the most important contributions a
+community member can make for a projects health. Stave Klabnik's
+[article][gardening] on how to be an open source gardener expresses our values.
+
+#### Making Bugs Actionable
+
+Bugs are labeled [`not-actionable`][na] when they are filed. Here is a checklist
+for making bugs actionable:
+
+1. Check for duplicate bugs
+2. Verify that the bug is reproducable
+3. Define steps to reproduce with expected and actual results
+4. Create a small test case. We have a [glitch] template for sharing examples.
```

Comment:
I think that there might also be a 5th step for "Categorize the issue report with a suitable tag" linking to kind of an overview on what you have ('needs-ux', 'performance'). FWIW, I cannot find such an overview. This could be just a section of contributing.md or - well, even better - in this maintainer.md ;)